### PR TITLE
Added extra options to the Bar and Pie Chart dialogs

### LINKED ISSUE #6726: Add lollipop option to the bar chart dialogue
We recently (I think) added geom_lollipop to the list.

This is a sort of bar chart.  It is (in geom terms) also a segment plus point.  But, as there is a geom lollipop I suggest the best place is in the bar chart dialogue.  It can be a single checkbox, on the left, perhaps after Polar.  If checked, then we use geom_lollipop rather than geom_bar, so the bar options button should change to Lollipop Options.
There should be 2 more controls on the same line as the Lollipop checkbox.  They are labelled Size, and Colour and use those controls from the Lollipop sub-dialogue.
This should be implemented for both the current options of the bar chart dialogue.

### PR DESCRIPTION
**Pull request by MeSophie**
Fixes #6726.
 I noticed that Part a) and b) was already done. 

image

image
 
@lloyddewit and @rdstern Please have a look.

### CODE REVIEW

**Comment by N-thony** [comment]
@anastasia-mbithe can you peer review this?

File: instat/dlgBarAndPieChart.vb [hunk 1]
```diff
@@ -1019,4 +1028,20 @@ Public Class dlgBarAndPieChart
...
+
+    End Sub
+
+    Private Sub ucrReceiverX_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverX.ControlContentsChanged
+
+    End Sub
+
+    Private Sub lblReorder_Click(sender As Object, e As EventArgs) Handles lblReorder.Click
+
+    End Sub
```
Comment (anastasia-mbithe) [inline instat/dlgBarAndPieChart.vb:1046]: @MeSophie, remove these empty subs

File: instat/dlgBarAndPieChart.vb [hunk 2]
```diff
@@ -718,13 +727,13 @@ Public Class dlgBarAndPieChart
...
             If ucrVariablesAsFactorForBarChart.bSingleVariable Then
-                ucrInputReorderX.Visible = True
+                'ucrInputReorderX.Visible = True
                 ucrInputAddReorder.Visible = Not ucrReceiverByFactor.IsEmpty()
                 If Not ucrInputAddReorder.Visible Then
                     ucrInputAddReorder.SetText(strNone)
                 End If
             Else
-                ucrInputReorderX.Visible = False
+                'ucrInputReorderX.Visible = False
```
Comment (anastasia-mbithe) [inline instat/dlgBarAndPieChart.vb:736]: Remove this line too

File: instat/dlgBarAndPieChart.vb [hunk 3]
```diff
@@ -718,13 +727,13 @@ Public Class dlgBarAndPieChart
         ucrChkLollipop.Enabled = If(rdoValue.Checked, True, False)
         If rdoFrequency.Checked Then
             If ucrVariablesAsFactorForBarChart.bSingleVariable Then
-                ucrInputReorderX.Visible = True
+                'ucrInputReorderX.Visible = True
```
Comment (anastasia-mbithe) [inline instat/dlgBarAndPieChart.vb:730]: Remove this line since its not used

**Comment by N-thony** [comment]
@anastasia-mbithe are you done reviewing this?

**Review by anastasia-mbithe — APPROVED** [review]

**Review by anastasia-mbithe — APPROVED** [review]

**Review by rdstern — APPROVED** [review]
Seems ok now

File: instat/dlgBarAndPieChart.vb [hunk 4]
```diff
@@ -314,6 +314,11 @@ Public Class dlgBarAndPieChart
         ucrChkReorderValue.AddFunctionNamesCondition(False, "reorder", False)
         ucrChkReorderValue.AddToLinkedControls(ucrInputReorderValue, {True}, bNewLinkedHideIfParameterMissing:=True)
 
+        ucrChkReorderFrequency.SetText("Rorder Frequency")
```
Comment (lloyddewit) [inline instat/dlgBarAndPieChart.vb:317]: Should this be:
```suggestion
        ucrChkReorderFrequency.SetText("Reorder Frequency")
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@MeSophie Thank you for the changes. I just have one small suggested change. I would have made it myself but I wanted to double-check with you that it's OK.
Thanks

**Comment by MeSophie** [comment]
> @MeSophie Thank you for the changes. I just have one small suggested change. I would have made it myself but I wanted to double-check with you that it's OK. Thanks

Thanks @lloyddewit . It's done.

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**